terraform/projects/app-ecs-services/templates/auth-proxy.conf.tpl

```diff
@@ -27,15 +27,56 @@ server {
   auth_basic_user_file /etc/nginx/conf.d/.htpasswd;
 
 
-  server_name ~(alert*);
+  server_name alerts-1.*;
 
+  set $alert "${alertmanager_1_dns_name}";
```

Comment:
why do we use `set` here rather than just proxying directly to `${alertmanager_1_dns_name}`?